src/blk/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-if(WITH_BLUESTORE OR WITH_RBD_RWL)
```

Comment:
Nit: does `WITH_RBD_RWL` actually use this file? or was that to support this SSD-cache work?

Reply:
WITH_RBD_RWL  doesn't use libblk_srcs, it is just for SSD-cache.

Reply:
RWL doesn't use it. Got left out in clean-up. Removed